File: designpatterns/src/com/renho/behavior/ChainofResponsibility/ConcreteHandlerC.java
package com.renho.behavior.ChainofResponsibility;

public class ConcreteHandlerC extends Handler {

	@Override
	public void handleRequest(int request) {
		if(request > 5) {
			System.out.println("²»ÔÊĞí");	
		}else {
			System.out.println("ConcreteHandlerC:½áÊø");			
		}
	}

}

File: ideaworkspace/designpatterns/src/main/scala/com/renho/behavior/chainofresponsibility/ConcreteHandlerC.scala
package com.renho.behavior.chainofresponsibility

/**
  * Created by renho on 2016/7/11.
  */
class ConcreteHandlerC extends Handler {
	override def handlerRequest(request: Int): Unit = {
		if(request > 5) {
			println("不允许");
		}else {
			println("ConcreteHandlerC:结束");
		}
	}
}
